### PR TITLE
test(e2e): service ingestion and schedule options

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/DataInsightReportApplication.spec.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        `/api/v1/apps?limit=15&include=non-deleted`
+      );
+
+      await settingClick(page, GlobalSettingOptions.APPLICATIONS);
+      await appsResponse;
+    });
+
+    test('Install application', async ({ page }) => {
+      await page.click('[data-testid="add-application"]');
+      // Verify response status code (this is done implicitly by the route fulfillment)
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/DataInsightReportApplication.spec.ts:41]: we can remove this!

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/DataInsightReportApplication.spec.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        '[data-testid="data-insights-report-application-card"] [data-testid="config-btn"]'
+      );
+      await page.click('[data-testid="install-application"]');
+      await page.click('[data-testid="save-button"]');
+      await page.click('[data-testid="submit-btn"]');
+      await page.click('[data-testid="cron-type"]');
+      await page.click('[data-value="5"]');
+      await page.click('[data-testid="deploy-button"]');
+
+      // Verify response status code (this is done implicitly by the route fulfillment)
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/DataInsightReportApplication.spec.ts:52]: Same

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceIngestion.spec.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import SupersetIngestionClass from '../../support/entity/ingestion/SupersetIngestionClass';
+import { redirectToHomePage } from '../../utils/common';
+import { settingClick } from '../../utils/sidebar';
+
+const services = [
+  S3IngestionClass,
+  MetabaseIngestionClass,
+  MysqlIngestionClass,
+  // Todo: need to skip for time being as AUT runs on argo, and airflow is not available
+  //   new AirflowIngestionClass(),
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceIngestion.spec.ts:33]: I think based on isOSS flag we can now enable disable this test? WDYT?

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceIngestion.spec.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  const service = new ServiceClass();
+
+  test.describe.serial(service.serviceType, { tag: '@ingestion' }, async () => {
+    test.beforeEach('Visit entity details page', async ({ page }) => {
+      await redirectToHomePage(page);
+      await settingClick(page, service.category);
+    });
+
+    test(`Create & Ingest ${service.serviceType} service`, async ({ page }) => {
+      test.slow(true);
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceIngestion.spec.ts:56]: in serial mode, i dont think we will need for test.slow

File: openmetadata-ui/src/main/resources/ui/playwright.config.ts [hunk 5]
```diff
@@ -69,7 +71,7 @@ export default defineConfig({
       use: { ...devices['Desktop Chrome'] },
       // Added admin setup as a dependency. This will authorize the page with an admin user before running the test. doc: https://playwright.dev/docs/auth#multiple-signed-in-roles
       dependencies: ['setup'],
-      grepInvert: /data-insight/,
+      grep: /ingestion/,
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/playwright.config.ts:74]: I will revert this before making this ready for review, it's just reducing number of spec runs

File: openmetadata-ui/src/main/resources/ui/playwright/constant/service.ts [hunk 6]
```diff
@@ -24,3 +25,17 @@ export const SERVICE_TYPE = {
   StoredProcedure: GlobalSettingOptions.STORED_PROCEDURES,
   ApiService: GlobalSettingOptions.APIS,
 };
+
+export const REDSHIFT = {
+  serviceType: 'Redshift',
+  serviceName: `redshift-ct-test-${uuid()}`,
+  tableName: 'boolean_test',
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/playwright/constant/service.ts]: Done thanks

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceIngestion.spec.ts (hunk 3 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/ServiceIngestion.spec.ts]: Done

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**